introduction-to-datasets.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: page
+title: Introduction to Datasets
+---
+
+A dataset is a collection of records, similar to a relational database table. Records are similar to table rows, but the columns can contain not only strings or numbers, but also nested data structures such as lists, maps, and other records.
+
+To define a dataset, Kite minimally requires a [URI](#uris) and a [schema](#schemas).
+
+## URIs
+
+Kite identifies datasets by URI. The URI you provide tells Kite how and where to store data. For example, a dataset created with the URI `dataset:hdfs:/user/cloudera/datasets/movies` is stored in HDFS.
+
+The default storage _scheme_ (as opposed to a _[schema](#schemas)_, described below) is Hive. If you omit the scheme in your URI, your dataset is stored in Hive. For example, the URI `movies` is equivalent to `dataset:hive:/movies`. A dataset you create with this URI is stored as a table in Hive's `default` database.
+
+Dataset URIs cannot contain periods (.).
+
+You can create datasets in Hive, HDFS, HBase, or as local files. See [Dataset and View URIs]({{site.baseurl}}/URIs.html).
+
+## Schemas
+
+A schema defines the field names and datatypes for a dataset. Kite relies on an Apache Avro schema definition for each dataset. For example, this is the schema definition for a table using four columns from the `movies` dataset.[<sup>1</sup>](#notes)
+
+```json
+{
+  "type":"record",
+  "name":"Movie",
+  "namespace":"org.kitesdk.examples.data",
+  "fields":[
+    {"name":"id","type":"int"},
+    {"name":"title","type":"string"},
+    {"name":"releaseDate","type":"string"},
+    {"name":"imdbUrl","type":"string"}
```

Comment:
Best practice is to use snake_case rather than camelCase for field names because there are reports of inconsistent handling of upper case letters. So this should be "release_date" and "imdb_url".